react/src/reactTextMask.js

```diff
@@ -1,41 +1,26 @@
-import React, {PropTypes} from 'react'
-import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
-
-export const MaskedInput = React.createClass({
-  propTypes: {
-    mask: PropTypes.oneOfType([
-      PropTypes.array,
-      PropTypes.func,
-      PropTypes.bool,
-      PropTypes.shape({
-        mask: PropTypes.oneOfType([PropTypes.array, PropTypes.func]),
-        pipe: PropTypes.func
-      })
-    ]).isRequired,
-    guide: PropTypes.bool,
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    pipe: PropTypes.func,
-    placeholderChar: PropTypes.string,
-    keepCharPositions: PropTypes.bool,
-    showMask: PropTypes.bool
-  },
-
-  createTextMaskInputElement,
```

Comment:
no it is not _needed_, true.. however, it was put there for a reason and I don't see the need to remove it